README.md

```diff
@@ -7,16 +7,21 @@
 
 [![R-CMD-check](https://github.com/a-b-street/abstr/workflows/R-CMD-check/badge.svg)](https://github.com/a-b-street/abstr/actions)
 <!-- badges: end -->
+<img src="https://user-images.githubusercontent.com/22789869/129973263-5fc74ae3-ed17-4155-9a8c-7f382f7796cc.png" align="left" height="164px" width="164px" margin="10%" />
 
 The goal of abstr is to provide an R interface to the [A/B
 Street](https://github.com/a-b-street/abstreet#ab-street) transport
 planning/simulation game. Currently it provides a way to convert
 aggregated origin-destination data, combined with data on buildings
 representing origin and destination locations, into `.json` files that
-can be directly imported into the A/B Street game. See [the formats page
-in the A/B Street
+can be directly imported into the A/B Street game. 
```

Comment:
Minor point for future reference (my bad): I think it's best to call it city simulation software than a game for comms but that does not affect this review, just thought it worth saying when I saw it.